config/dbupdates.php

```diff
@@ -40,4 +40,123 @@
 */
 
 
+$system__database_upgrades[]=array(
+'version'=>'2020021200', 
```

Comment:
May be it worth indenting all items in array? i.e
```
$system__database_upgrades[]=array(
    'version'=>'2020021200', 
    'type'=>'new_admin_right',
    'specs'=> array(
        'right_name'=>'pform_anonymization_fields_edit', 
        'admin_types'=>array('admin','developer','installer'),
    )
);
```

Reply:
Ok, done.